### ISSUE TITLE
Canvas fails to render on KDE

### ISSUE THREAD

**Issue by DominikNovosel** [astrofox-io/astrofox#34]
System information:
Operating System: openSUSE Leap 15.2
KDE Plasma Version: 5.18.6
KDE Frameworks Version: 5.71.0
Qt Version: 5.12.7
Kernel Version: 5.3.18-lp152.60-default
OS Type: 64-bit
Processors: 4 × Intel® Core™ i5-3230M CPU @ 2.60GHz
Memory: 7.6 GiB of RAM

Running from 1.1.0. AppImage. Upon startup, the canvas surface is rendered as blank and gray, despite being set to black by default and including some starter elements. Changing this parameter has no effect. Even after importing some audio and an image, the canvas remains blank and unresponsive. Resizing the canvas causes an icon to render in the top-left corner. I tried running Storyboarder to check for issues with my graphics drivers or Electron, but it worked fine. Three.js reported several errors. Any idea what's going on? The full console log and screenshots should be below.

[-1611681812783.log](https://github.com/astrofox-io/astrofox/files/5875243/-1611681812783.log)
![Screenshot_1](https://user-images.githubusercontent.com/66269443/105887476-14d04680-600c-11eb-805d-5c56bd08910b.png)
![Screenshot_2](https://user-images.githubusercontent.com/66269443/105887479-1568dd00-600c-11eb-92d8-854b5766eefc.png)

**Comment by mikecao** [comment]
Looks like it's failing to get the webgl context. Can you see if this site works for you? https://get.webgl.org/webgl2/

**Comment by DominikNovosel** [comment]
Yup, it seems to work just fine. At least now I know my drivers are alright.
![Screenshot_3](https://user-images.githubusercontent.com/66269443/105953021-c7d38b00-6072-11eb-9887-b7746aafbd92.png)

**Comment by mikecao** [comment]
Looks like you're running Firefox. Electron is Chromium based, though I'm not sure if that would make a difference. Just wanted to point it out.

**Comment by DominikNovosel** [comment]
I was curious myself, so I tried WebKit and Chromium too, they worked as well.
![Screenshot_20210315_190924](https://user-images.githubusercontent.com/66269443/111201510-21602c80-85c3-11eb-9025-2a6d89480815.png)
![Screenshot_20210315_191048](https://user-images.githubusercontent.com/66269443/111201517-22915980-85c3-11eb-9025-0085cbc150eb.png)

**Comment by connervieira** [comment]
For what it's worth, I'm on Pop!_OS Linux (Debian/Ubuntu based) running GNOME, and my canvas seems to render just fine with the AppImage of Astrofox. It might have something to do with your specified distro.

**Comment by DominikNovosel** [comment]
I recently switched to:
Operating System: Kubuntu 20.04
KDE Plasma Version: 5.18.5
KDE Frameworks Version: 5.68.0
Qt Version: 5.12.8
Kernel Version: 5.11.0-43-generic
OS Type: 64-bit
Processors: 4 × Intel® Core™ i5-3230M CPU @ 2.60GHz
Memory: 7.6 GiB of RAM

and I can unfortunately confirm that this issue persists in Astrofox 1.3.0. I think the issue is KDE-specific, since the aforementioned icon is the standard for denoting errors in Plasma Workspace.

**Comment by localesp** [comment]
I have got the same problem on Manjaro i3 or Manjaro GNOME.

OS: ManjaroLinux (21.3.2)
Kernel: 5.9.16-1-MANJARO
CPU: Intel(R) Core(TM) i3-3220 CPU @ 3.30GHz

**Comment by mikecao** [comment]
This was reported on 1.1.0 with a much older Electron/three.js. The rendering stack was completely rewritten for 2.0.

Closing as resolved in Astrofox 2.0. If you can still reproduce this on 2.x, please open a new issue with steps and your version.
